src/chartserver/handler_manipulation.go

```diff
@@ -69,7 +69,7 @@ func (c *Controller) DeleteChartVersion(namespace, chartName, version string) er
 		return errors.New("invalid chart for deleting")
 	}
 
-	url := fmt.Sprintf("/api/chartrepo/%s/charts/%s/%s", namespace, chartName, version)
+	url := fmt.Sprintf("/api/v2.0/chartrepo/%s/charts/%s/%s", namespace, chartName, version)
```

Comment:
How about define a const for the API version as we need to change it when bump up the API version again?